plugins/com.gwtplugins.gdt.eclipse.core/src/com/google/gdt/eclipse/core/resources/ProjectResources.java

```diff
@@ -128,13 +128,13 @@ public static List<File> findFilesInDir(File directory, FilenameFilter filter) {
   /**
    * Given a java.io.File containing Java source, call the Eclipse auto-format
    * code on that source and write it back to disk.
-   * 
+   *
    * @param file
    * @throws CoreException
    */
   public static void reformatJavaSource(File file) throws CoreException {
     try {
-      String generatedSource = textFromFile(file);
+      String generatedSource = Files.readString(file.toPath(), Charset.defaultCharset());
```

Comment:
consider if we really want the default charset here, or just assume utf8?

Reply:
This has been default charset in the previous implementation. Didn't want to change that. Just simplified the Code by using an existing JRE method.